integration-tests/playbook_verifier/test_verifier.py

```diff
@@ -19,14 +29,23 @@
         "compliance_openscap_setup.yml",
     ],
 )
+@pytest.mark.tier1
 def test_official_playbook(filename: str):
-    """insights-client contains playbook verifier application.
-
-    It is used by rhc-worker-playbook and rhc-worker-script to safely deliver
-    and run Red Hat signed playbooks.
-
-    In this test, the official playbooks are verified against the GPG key
-    the application ships.
+    """
+    :id: 3659e27f-3621-4591-b1c4-b5f0a277bb72
+    :title: Test playbook verifier
+    :description:
+        In this test, the official playbooks are verified against the GPG key
+        the application ships.
```

Comment:
Just a simple rewording:
```suggestion
    :description:
        This test verifies the official playbooks against the GPG key
        the application ships.
```

Reply:
Still applies